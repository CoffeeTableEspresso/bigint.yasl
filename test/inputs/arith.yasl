
const bigint = __require_c__('bigint')

const k = bigint.bigint('111111111111')
const n = bigint.bigint('222222222222')

echo k
echo n

echo -n
echo +n

echo n * k
echo n // k
echo n % k

echo n + k
echo n - k

echo n & k
echo n | k
echo n ^ k
