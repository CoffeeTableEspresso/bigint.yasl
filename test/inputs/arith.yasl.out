111111111111
222222222222
-222222222222
222222222222
24691358024641975308642
2
0
333333333333
111111111111
75635360134
257697973199
182062613065
